db/models.py

```diff
@@ -1 +1,27 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(Race, on_delete=models.CASCADE)
```

Comment:
According to the checklist, you should not confuse the functionality of the `on_delete` parameter. You are using `on_delete=models.CASCADE` which means when the referenced object is deleted, also delete the objects that have references to it. If it is the intended behavior then it's fine. Otherwise, consider using `on_delete=models.SET_NULL` if you know that the object should not be deleted, even if the object it references may be deleted.